mllib/src/test/scala/org/apache/spark/ml/util/GBTSuiteHelper.scala

```diff
@@ -0,0 +1,271 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.ml.util
+
+import scala.collection.mutable.ArrayBuffer
+
+import org.scalactic.TolerantNumerics
+
+import org.apache.spark.SparkFunSuite
+import org.apache.spark.ml.classification.GBTClassifier
+import org.apache.spark.ml.feature.LabeledPoint
+import org.apache.spark.ml.linalg._
+import org.apache.spark.ml.regression.GBTRegressor
+import org.apache.spark.ml.tree.{InternalNode, LeafNode, Node}
+import org.apache.spark.mllib.tree.impurity.{ImpurityAggregator, ImpurityCalculator}
+import org.apache.spark.mllib.tree.loss.Loss
+import org.apache.spark.sql._
+import org.apache.spark.sql.functions._
+
+object GBTSuiteHelper extends SparkFunSuite {
+  implicit val approxEquals = TolerantNumerics.tolerantDoubleEquality(1e-3)
+
+  /**
+   * @param labels set of GBT labels
+   * @param agg the aggregator to use
+   * @return the calculator from aggregation on the labels
+   */
+  def computeCalculator(labels: Seq[Double],
```

Comment:
style: first argument goes on next line for multi-line method declarations
(same issue elsewhere)